### PR TITLE
Fixes Documentation issue #245

### LINKED ISSUE #245: Document that the `passive_declares` config value is mandatory on Fedora's private vhost
It's a common mistakes for users of Fedora's `/pubsub` vhost to forget to set `passive_declares` to `true`, but it's apparently not documented here.

### PR DESCRIPTION
**Pull request by vidit-maheshwari**
### Summary
Fixes #245 
This pull request aims to improve the documentation for configuring Fedora Messaging by adding a reminder regarding the `passive_declares` option to be set `true`.

### Changes Made
- Added a reminder note to the README file emphasizing the importance of setting `passive_declares` to `true` in the Fedora Messaging configuration.
- Provided clear instructions for users to locate the `config.toml` file and set the `passive_declares` option.

### Purpose
By enhancing the documentation with this reminder, we aim to reduce potential issues for users of Fedora's `/pubsub` vhost who might forget to configure `passive_declares`. Ensuring that `passive_declares` is correctly set to `true` is essential for proper functioning of queues and exchanges in Fedora Messaging.

Here is the preview of the changes made:

image

### CODE REVIEW

File: docs/tutorial/installation.rst [hunk 1]
```diff
@@ -67,3 +67,17 @@ Comment out the ``callback`` and ``bindings`` options, and all the
 
 In the ``[client_properties]`` section, change the ``app`` value to ``Fedora
 Messaging tutorial``.
+
+
+Note: It is essential to configure the `passive_declares` option correctly in the `etc/fedora-messaging/config.toml` file. This setting is mandatory for users of Fedora's `/pubsub` vhost and should be set to `true`. It controls the declaration of queues and exchanges.
+
+Ensure the following steps are taken to configure `passive_declares`:
+
+1. **Access Configuration File**: Locate the `config.toml` file for Fedora Messaging. This file is typically found at `config.toml.example`.
```
Comment (abompard) [inline docs/tutorial/installation.rst]: That part is not correct, applications will usually have their Fedora Messaging config file at `/etc/fedora-messaging/config.toml". The `.example` file is just what is provided by the library as an example, that's not what client applications will actually use.

Comment (vidit-maheshwari) [reply docs/tutorial/installation.rst]: Is it fine now ?

**Comment by vidit-maheshwari** [comment]
@abompard i have made the necessary changes, could you please review ?

**Comment by abompard** [comment]
This setting is actually only necessary for clients of Fedora's restricted broker. Please add it to the appropriate section at the bottom of `docs/user-guide/quick-start.rst`.

**Comment by vidit-maheshwari** [comment]
@abompard i have done the required changes as per your instructions. Kindly review it.

File: docs/user-guide/quick-start.rst [hunk 2]
```diff
@@ -145,3 +145,11 @@ Connecting the Fedora's private virtual host requires working with the Fedora
 infrastructure team. The current process and configuration for this is
 documented in the `infrastructure team's development guide
 <https://fedora-infra-docs.readthedocs.io/en/latest/dev-guide/messaging.html>`_.
+
+**Note:** It is essential to configure the ``passive_declares`` option correctly in the ``/etc/fedora-messaging/config.toml`` file. This setting is mandatory for users of Fedora's ``/pubsub`` vhost and should be set to ``true``. It controls the declaration of queues and exchanges.
+
+Example:
```
Comment (abompard) [inline docs/user-guide/quick-start.rst]: Given the text above, I don't think this example brings a lot of value.

File: docs/user-guide/quick-start.rst [hunk 3]
```diff
@@ -145,3 +145,11 @@ Connecting the Fedora's private virtual host requires working with the Fedora
 infrastructure team. The current process and configuration for this is
 documented in the `infrastructure team's development guide
 <https://fedora-infra-docs.readthedocs.io/en/latest/dev-guide/messaging.html>`_.
+
+**Note:** It is essential to configure the ``passive_declares`` option correctly in the ``/etc/fedora-messaging/config.toml`` file. This setting is mandatory for users of Fedora's ``/pubsub`` vhost and should be set to ``true``. It controls the declaration of queues and exchanges.
```
Comment (abompard) [inline docs/user-guide/quick-start.rst]: Please follow the line length convention of this file.

**Review by abompard — CHANGES_REQUESTED** [review]

**Comment by vidit-maheshwari** [comment]
@abompard I have made the changes according to the line length convention and removed the example section as you said, Kindly review it if further any changes are required.

File: docs/user-guide/quick-start.rst (hunk 3 above)
Comment (vidit-maheshwari) [reply docs/user-guide/quick-start.rst]: Hey @abompard i have made the changes kindly review it.

**Review by abompard — APPROVED** [review]
Looks good, thanks!

**Comment by abompard** [comment]
Ah there's just one small thing, we have a linter that detected something wrong with the files not ending in a newline. Please setup pre-commit: `dnf install pre-commit; pre-commit install` and run the tox check named `checks`: `tox -e checks`. This should show you the files needing a fix. Then you can amend your commit: `git commit -a --amend` and push again to your PR: `git push --force`.

**Pull request closed without merging**